composer.json

```diff
@@ -57,7 +58,13 @@
       "Tests\\EoneoPay\\Externals\\": "tests/"
     }
   },
+  "extra": {
```

Comment:
I think this should be removed now that composer.lock is up to date

Reply:
Removing this drops the `eonx-com/search` version back down to `2.1.7`, so this will have to remain in place.